minibot/hardware/fake/gpio.py

```diff
@@ -0,0 +1,90 @@
+"""
+Minibot GPIO classes.
```

Comment:
Add more comments about why this is in the `fake` folder (also maybe call it something else, like `virtual`?).